examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -2143,6 +2143,8 @@ void setup(void) {
   if (mdns.begin(Hostname)) {
 #endif  // ESP8266
     debug("MDNS responder started");
+    // Announce esp tcp service on port 8080
+    mdns.addService("esp", "tcp", 8080);
```

Comment:
Use `kHttpPort` instead of `8080` and `"http"` rather then `"esp"`.
Adjust code comment to match.

Reply:
OK, make sense.